client-gateway/bot/drive_ops_bot.py

```
@@ -0,0 +1,151 @@
+    
+    bot.send_message(chat_id, summary, parse_mode='Markdown', reply_markup=markup)
+
+# --- Обробка кнопок підтвердження ---
+
+@bot.callback_query_handler(func=lambda call: call.data.startswith('order_'))
+def handle_order_status(call):
+    if call.data == "order_confirm":
+        bot.edit_message_text(
+            chat_id=call.message.chat.id,
+            message_id=call.message.message_id,
+            text="✅ **Замовлення прийнято!**\nШукаємо найближче авто...",
```

Ruff (0.14.10)

String contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF001)